deploy.yml

```
@@ -0,0 +1,36 @@
+name: Deploy to GitHub Pages
+
+on:
+  push:
+    branches:
+      - main
+
+permissions:
+  contents: write
+
+jobs:
+  deploy:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+      
+      - name: Install dependencies
+        run: npm install
+
+      - name: Build project
+        run: npm run build
+
+      - name: Set up Deno
+        uses: denoland/setup-deno@v1
+        with:
+          deno-version: v1.x  
+
+      - name: Build project
+        run: deno run --allow-read --allow-write build.ts

```

```suggestion
      - name: Set up Deno
        uses: denoland/setup-deno@v2
        with:
          deno-version: v2.x  
      - name: Install dependencies
        run: deno install
      - name: Build project
        run: deno task build
```
Actualizada la version de Deno para ser la ultima, reordenado la ejecucion, el paso de build project eliminado no es necesario ya que `deno task build` realiza esto mismo y conforme al uso correcto de Deno se sustituyo por `deno install` y `deno task build`